### PR TITLE
Deploy to production

### PR DESCRIPTION
**Pull request by Greenheart**
## Included changes

- Show map view first for municipalities
- Remove duplicated words in company data
- Remove unused Borna font assets
- Improved table UX, scrolling the full page, sticky table header and table footer. Improved layout on smaller screen sizes.
- Updated tests for SMHI data
- Removed old data output files and move consumption data to Excel
- Feature: Make Python tests for SMHI data independent from the last year with data
- Also added Instagram logo and link to footer

### CODE REVIEW

**Comment by Greenheart** [comment]
As soon as https://github.com/Klimatbyran/klimatkollen/pull/602 is merged, we can test and deploy this.

File: data/output/climate-data.json
Comment (elvbom) [inline data/output/climate-data.json:1]: This data is incorrect. I'll fix in a new PR which should be included in this release

Comment (Greenheart) [reply data/output/climate-data.json:1]: Ok, good catch!

**Review by elvbom — CHANGES_REQUESTED** [review]
The table footer font size is rather small on desktop, can it be made larger?

Screenshot 2024-05-29 at 10 51 20

**Review by elvbom — APPROVED** [review]

**Pull request merged**